### PR TITLE
8comments

### PR DESCRIPTION
**Pull request by Asileka**

### CODE REVIEW

**Comment by douglashellowell** [comment]
Nice work!

---

At the moment if there are no comments the users see a empty space - this could be confusing! It would be nice to render a message (something like "There are no comments on this review")

---

You should format your date string to be more human readable - users won't know what `2021-03-27T14:15:38.110Z` means!

---

**Comment by douglashellowell** [comment]
In HTML the **first child** of a `<ul>`/`<ol>` should be a `<li>` - there are places (see `Review.jsx`) where you have other elements as children of the `<ul>`

Either make sure you're using `<li>` - or reconsider this component is a list (I don't think a single Review is a list...)

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+const Comments = () => {
+  const { review_id } = useParams();
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getComments(review_id).then((comments) => {
+      setComments(comments);
+      console.log(comments);
```
Comment (douglashellowell) [inline src/components/Comments.jsx]: make sure you are removing your console.logs before commiting!

**Comment by douglashellowell** [comment]
You should have a route for `/`

Whether this is a welcome message for your app or something else is up to you!

**Review by Angaca — APPROVED** [review]
Good clean up, ready to go

**Pull request merged**